calliope/config/defaults.yaml

```diff
@@ -85,8 +85,7 @@ techs:
                 om_var: 0  # Variable O&M costs (per kWh of ``es_prod``)
                 om_fuel: 0  # Fuel costs (per kWh of ``r`` used)
                 om_rb: 0  # Fuel costs for secondary resource (per kWh of ``rb`` used)
-        revenue:
-            default:
+                ## Revenue generating subsidies for production
                 sub_var: 0 #variable subsidy (per kWh of ``es_prod`` or ``es_con``)
```

Comment:
Yeah, I'd be happy with that: `rev_var`, `rev_cap`, `rev_annual`, and `rev_export` (if that ends up being included)